ILSpy/ILSpy.csproj

```diff
@@ -844,7 +844,14 @@
   </PropertyGroup>
 
   <Import Project="$(VCToolsVersionPropsFile)" Condition="Exists('$(VCToolsVersionPropsFile)')" />
-  
+
+  <Target Name="SortResX" BeforeTargets="BeforeBuild">
```

Comment:
@dgrunwald I hope you are OK with these changes. The purpose of the script is to make sure that all resx files are sorted by resource string key on build to minimize the probability of merge conflicts in the future.